alteryx/featuretools
# v1.13.0

**v1.13.0 Aug 18, 2022**
* Enhancements
* Fixes
    * Allow boolean columns to be included in remove_highly_correlated_features (#2231)
* Changes
    * Refactor schema version checking to use `packaging` method (#2230)
    * Extract duplicated logic for Rolling primitives into a general utility function (#2218)
    * Set pandas version to >=1.4.0 (:pr:`2246`)
    * Remove workaround in `roll_series_with_gap` caused by pandas version < 1.4.0 (#2246)
* Documentation Changes
    * Add line breaks between sections of IsFederalHoliday primitive docstring (#2235)
* Testing Changes
    * Update create feedstock PR forked repo to use (#2223, #2237)
    * Update development requirements and use latest for documentation (#2225)

-----
*After creating the pull request: in order to pass the **release_notes_updated** check you will need to update the "Future Release" section of* `docs/source/release_notes.rst` *to include this pull request.*
